### ISSUE TITLE
Autoscaling of merged tracks does not follow the current view of the region

### ISSUE THREAD

**Issue by hoholee** [igvteam/igv.js#1506]
Hi IGV team,

I was trying to make merged tracks with autoscale (related to #1473) using the following code (I'm using `igv-webapp.1.10.8` and I believe it loads the latest igv.js `2.12.6`):

```
                    {
                    "name": "Merged - autoscaled",
                    "type": "merged",
                    "autoscale": true,
		    "height": 50,
                    "tracks": [
                        {
                            "type": "wig",
                            "format": "bigwig",
                            "url": "https://www.encodeproject.org/files/ENCFF000ASJ/@@download/ENCFF000ASJ.bigWig",
                            "color": "red"
                        },
                        {
                            "type": "wig",
                            "format": "bigwig",
                            "url": "https://www.encodeproject.org/files/ENCFF351WPV/@@download/ENCFF351WPV.bigWig",
                            "color": "green"
                        }
                    ]
                },
                    {
                    "name": "track1",
                    "type": "wig",
                    "format": "bigwig",
		    "url": "https://www.encodeproject.org/files/ENCFF000ASJ/@@download/ENCFF000ASJ.bigWig",
		    "color": "red",
		    "height": 50,
                    "autoscale": true,
                    "autoscaleGroup": "group1"
                },
                    {
                    "name": "track2",
                    "type": "wig",
		    "format": "bigwig",
		    "url": "https://www.encodeproject.org/files/ENCFF351WPV/@@download/ENCFF351WPV.bigWig",
		    "color": "green",
		    "height": 50,
                    "autoscaleGroup": "group1"
                }
```

It seems to be working OK at this locus:
![image](https://user-images.githubusercontent.com/10512707/167790586-2a829011-cac2-4c0a-a86d-8f8ee34d3445.png)

However, when I dragged the view a bit to the left I got this:
![image](https://user-images.githubusercontent.com/10512707/167790949-ccaaeef3-576d-4084-92ec-7564a3b65977.png)

The two constitutive tracks autoscaled correctly (since the rightmost highest peaks are now out of view), but the merged tracks did not.

This seems to be something wrong in the merged track type where the y-axis range is not computed based on the values in the browser's current view (but somehow extended further to both sides)? Or am I missing something here?

**Comment by jrobinso** [comment]
Yes I agree that seems to be a bug.   I will look at it asap.

**Comment by julianstanley** [comment]
https://user-images.githubusercontent.com/22749289/168898183-0224f89b-d9b2-4eb0-96a1-8d562fae45ed.mov

Just adding an additional demonstration of the same problem on the browser mentioned in #1473 after adding the `'autoscale': true` flag, [live demo here](https://web.mit.edu/julianst/www/rend/browsers/test.html)

**Comment by julianstanley** [comment]
> Yes I agree that seems to be a bug. I will look at it asap.

Any update on this / hints on what's happening here? Happy to try to help if there's anything I can do.

**Comment by jrobinso** [comment]
This is fixed, available in the main branch, and will be released by the end of the month.

**Comment by jrobinso** [comment]
Released as 2.13.4
